community/en-us/development/document.md

```diff
@@ -22,11 +22,13 @@ Make sure that your node version is 10+, docsite does not yet support versions h
 
 1. Run `npm install` in the root directory to install the dependencies.
 
-2. Run `npm run start` in the root directory to start a local server, you will see the website in 'http://localhost:8080'.
+2. Run commands to collect resources 2.1. Run export PROTOCOL_MODE=ssh tells Git clone resource via SSH protocol instead of HTTPS protocol 2.2. Run ./scripts/prepare_docs.sh prepare all related resources, for more information you could see how prepare script work.
```

Comment:
Enclose `export PROTOCOL_MODE=ssh` and `./scripts/prepare_docs.sh` in backticks (`) to denote a word as code like the [Build Website Locally](https://github.com/apache/dolphinscheduler-website/blob/master/README.md#build-website-locally). And add the link to [how prepare script work](https://github.com/apache/dolphinscheduler-website/blob/master/HOW_PREPARE_WOKR.md)